sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Python 3.9 too?

Reply:
Not supported by `dev-python/configshell-fb` or `dev-python/rtslib-fb`.

Reply:
It's fine for now, but can you make sure bugs are filed for those and/or add it yourself if tests pass for those?

Reply:
ping